jetty-maven-plugin/src/test/java/org/eclipse/jetty/maven/plugin/it/IntegrationTestGetContent.java

```diff
@@ -61,6 +61,7 @@ public void getContentResponse()
             }
             String contentCheck = System.getProperty("contentCheck");
             String pathToCheck = System.getProperty("pathToCheck");
+            System.out.println("contentCheck: " + contentCheck);
```

Comment:
Remove System.out

Reply:
this class has few more. 
And you will see them only when using m-invoker-p in debug mode or streamLogs which is not the case per default but only when debugging or when an IT test fail.